{{cookiecutter.project_slug}}/config/settings/production.py

```diff
@@ -11,16 +12,16 @@
 
 # DATABASES
 # ------------------------------------------------------------------------------
-DATABASES['default'] = env.db('DATABASE_URL')
-DATABASES['default']['ATOMIC_REQUESTS'] = True
-DATABASES['default']['CONN_MAX_AGE'] = env.int('CONN_MAX_AGE', default=60)
+DATABASES['default'] = env.db('DATABASE_URL')  # noqa F405
```

Comment:
Didn't `from .base import env` solved the issue with '`env` might be undefined'?

Reply:
This is complaint for `DATABASES`

Reply:
Oh, OK.